src/main/java/org/jabref/model/entry/BibEntry.java

```diff
@@ -603,10 +604,15 @@ public boolean allFieldsPresent(Collection<OrFields> fields, BibDatabase databas
     /**
      * Returns a clone of this entry. Useful for copying.
      * This will set a new ID for the cloned entry to be able to distinguish both copies.
+     * Ensures that the changed flag is set when the entry type equals the default entry type Misc
      */
     @Override
     public Object clone() {
         BibEntry clone = new BibEntry(type.getValue());
+        if (StandardEntryType.Misc.equals(type.getValue())) {
+            clone.changed = true;
```

Comment:
I would doubt that this fixed the issue for the other types. -- When the entry type is Article, the clone is unchanged, too. Thus, no serialization written

Reply:
New BibEntryType(entryType) always calls setType() and thus:
The default value for field.get is the default entry type misc.

https://github.com/JabRef/jabref/blob/7cc57474b7aa991ef6ea148d4030d4bb68ee69bf/src/main/java/org/jabref/model/entry/BibEntry.java#L360-L371

Reply:
For all other non default entry types the change flat is correclty set that was what this issue made so special.

Reply:
I wouldn't update the `changed` flag. The clone method should really clone the original entry, without any modifications (otherwise its not a clone).

However, the aux importer should set the changed flag of all entries (regardless of type), so that they get properly reformatted when writing them to disc (i.e. not using the parsed serialization of the original bib file).

Reply:
Well, the clone method implicitly sets the changed flags for all entry types except the default entry type. 
The aux importer has nothing to do with this changed or serialization stuff. It's only relevant for saving the newly created library.  I don't see any reason to add this logic to the AuxImporter
Only the BibTexParser does it.